distribution/docs/src/main/resources/content/_tables/CachingFederationStrategy.adoc

```diff
@@ -37,11 +37,11 @@
 |10080
 |true
 
-|Cache Everything
-|cachingEverything
-|Boolean
-|Cache all results unless configured as native
-|false
+|Query Result Cache Strategy
+|cacheStrategy
+|String
+|Strategy for caching query results. Valid entries are ALL, FEDERATED, and NONE.
```

Comment:
We should add the second sentence to the metatype.xml file that this table maps too

Reply:
I'll leave it out if that's okay so I don't have to pull down this branch 😂

Reply:
👍